src/pages/Search.jsx

```diff
@@ -1,9 +1,61 @@
-import ChannelListItem from "@/components/ChannelListItem";
+import { useEffect, useState } from "react";
+
+import SearchResultListItem from "@/components/SearchResultListItem";
+import { useChannelStore } from "@/store/useChannelStore";
+import { useSearchStore } from "@/store/useSearchStore";
 
 const Search = () => {
+  const radioChannelList = useChannelStore((state) => state.radioChannelList);
+  const interestChannelIds = useChannelStore(
+    (state) => state.interestChannelIds
+  );
+  const keyword = useSearchStore((state) => state.keyword);
+  const [filteredList, setFilteredList] = useState([]);
+
+  useEffect(() => {
+    const upperKeyword = keyword.toUpperCase().trim();
+
+    if (!upperKeyword) {
+      setFilteredList([]);
+      return;
+    }
+
+    const result = radioChannelList.filter(({ name }) =>
+      name.includes(upperKeyword)
```

Comment:
```suggestion
      name.toUpperCase().includes(upperKeyword)
```
대소문자 구분을 위해 `upperKeyword`를 사용해주셨네요!
그렇다면 비교를 위해 `name`도 `toUpperCase()`를 사용해야 할 것 같아요~

Reply:
해당 코드에 `toUpperCase()`를 사용하지 않은 이유는 저희 서비스에 라디오 채널명은 소문자가 존재하지 않아 불필요하다고 판단하여 넣지 않았는데요. 이 부분에 대해 희주님은 어떻게 생각하시나요?

Reply:
좋은 리뷰 감사드립니다! MBC, KBS, SBS 라디오 채널명만 생각하여 라디오 채널명은 모두 대문자일 줄 알았는데 iFM 경인방송과 같은 소문자 채널도 있었네요. 해당 내용 반영하였습니다!